types/react-native/index.d.ts

```diff
@@ -9591,7 +9591,8 @@ interface NativeModulesStatic {
  * Define lazy getters for each module. These will return the module if already loaded, or load it if not.
  * See https://reactnative.dev/docs/native-modules-ios
  * Use:
```

Comment:
Isn't this now obsolete with the explicit `@example`?

Reply:
Yes, it was and I have now removed it:

<img width="735" alt="Screen Shot 2021-12-10 at 16 14 24" src="https://user-images.githubusercontent.com/34488374/145648456-63be9791-97af-4ab5-885d-8a2da1008577.png">